main.py

```
@@ -86,13 +98,20 @@ def get_database_uri():
 db.init_app(app)
-with app.app_context():
-    try:
-        db.create_all()
-    except Exception as e:
-        logging.warning(f"Could not create database tables: {e}")
+migrate.init_app(app, db)
+if should_auto_create_tables(app.config["SQLALCHEMY_DATABASE_URI"]):
+    with app.app_context():
+        try:
+            db.create_all()
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)